doc/modules/metrics.rst

```diff
@@ -133,6 +133,20 @@ between two vectors. This kernel is defined as:
 where ``x`` and ``y`` are the input vectors. If :math:`\gamma = \sigma^{-2}`
 the kernel is known as the Gaussian kernel of variance :math:`\sigma^2`.
 
+.. _laplacian_kernel:
+
+Laplacian kernel
+----------------
+The function :func:`laplacian_kernel` is a variant on the radial basis 
+function kernel defined as:
+
+.. math::
+
+    k(x, y) = \exp( -\gamma \| x-y \|_1)
+
+where ``x`` and ``y`` are the input vectors and :math:`\|x-y\|_1` is the 
+Manhattan distance between the input vectors.
```

Comment:
I don't think there is a canonical simple case. It's the kernel of choice in the field of ML applied to molecular systems. The simplest example is probably in the tutorial review I referenced earlier. I will include it as a comment in the documentation.